stylus/components/alerts.styl

```diff
@@ -7,61 +7,68 @@
  * This file contains all needs for alerts
  */
 
-$alerts
-    alert-width = 40rem
-    .coz-alert
-        position          fixed
-        z-index           $alert-index-mobile
-        top               auto
-        right             0
-        bottom            3rem
-        left              0
-        display           flex
-        justify-content   space-between
-        align-items       center
-        box-sizing        border-box
-        box-shadow        0 .375rem 1.125rem 0 rgba(50, 54, 63, .23)
-        background-color  emerald
-        padding           .8rem 1rem
-        color             white
-        opacity           1
-        transition        transform .5s ease-out, opacity .2s ease-out
-        cursor            pointer
+alert-width = 40rem
 
-        p
-            margin      0
-            line-height 1.5
+$alert
+    position fixed
+    z-index $alert-index-mobile
+    top auto
```

Comment:
the default value is already auto, isn't it ?

Reply:
The spec says yes, but I remember some browsers have/had `0` for a default value, but maybe I'm too careful and there no need for this anymore. I'll check.